history/peter-toma.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Developer of the first machine translation product
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
+In 1956, Toma developed software for machine translation at the California Institute of Technology. 
+The system translated Russian texts into English. Later, he worked at Georgetown University where the system 
+Serna was developed for the same translation direction. 
+
+In 1962, he founded a company which created the systems Autotran and Technotran. 
```

Comment:
What was the name of the company?

Reply:
The article says "his own company". It doesn't give a name, and I couldn't find it.

Reply:
Maybe we should create Issues for these types of things?